app/build.gradle

```diff
@@ -48,6 +48,7 @@ dependencies {
     implementation 'androidx.cardview:cardview:1.0.0'
     implementation 'androidx.recyclerview:recyclerview:1.1.0-rc01'
     implementation 'com.jakewharton:butterknife:10.1.0'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
```

Comment:
We don't need constraint layout for this. I try to avoid constraint layout since I find its API to be confusing.